dals/issueFunctions.js

```diff
@@ -1,80 +1,104 @@
 const mongoose = require('mongoose');
 const Issue = mongoose.model('Issue');
 
-exports.create = (req, res) => {
-    return Issue.create({
-        title: req.payload.title,
-        description: req.payload.description,
-        name: req.payload.name
-    }, (err, data) => {
-        if(err){
-            res.response(err).code(500);
-        }
-        return res.response(data);
+exports.create = (issue) => {
+
+    return new Promise((resolve, reject) => {
+        Issue.create({
```

Comment:
Consult with mongoose to see whether the .create function returns the promise itself